htdocs_symfony/src/Entity/CachesEntity.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Oc\Entity;
+
+use Oc\Repository\AbstractEntity;
+//use Doctrine\ORM\Mapping as ORM;
+
+///**
+// * @ORM\Entity(repositoryClass="Oc\Repository\CachesRepository")
```

Comment:
Mehrzeilige DocBlocks sind im Format:
```suggestion
/**
 * @ORM\Entity(repositoryClass="Oc\Repository\CachesRepository")
 */
```

Reply:
das war ein mit Absicht auskommentierter Block, um zu schauen, ob ich ihn brauche. Ist jetzt aber raus.